src/main/java/frc/robot/constants/IntakeConstants.java

```diff
@@ -0,0 +1,8 @@
+package frc.robot.constants;
+
+public class IntakeConstants {
```

Comment:
Fix this issue https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&pullRequest=4&id=packofparts_frc2024-post-season&open=AY9LrfmENln50TcTJzdI&tab=code